src/demo/app/datepicker/daterange-demo.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'datepicker-demo',
+  template: `
+    <section class="main demo">
+      <h1>Date Range</h1>
+      <p>The prx-daterange component is a date range picker with
+        "from" and a "to" dates represented by a pair of <code>prx-datepicker</code>s.
+      </p>
+      <dl>
+        <dt>module</dt><dd><code>DatepickerModule</code></dd>
+        <dt>selector</dt><dd><code>prx-daterange</code></dd>
+      </dl>
+      <ul>
+        <li>
+          <code>@Input() format: string</code> (optional) sets the datepicker format shown in the form fields,
+          defaults to YYYY-MM-DD
+        </li>
+        <li>
+          <code>@Input() from: Date</code> (optional) sets "from" date
+        </li>
+        <li>
+          <code>@Input() to: Date</code> (optional) sets "to" date
+        </li>
+        <li>
+          <code>@Output() rangeChange: EventEmitter&lt;&#123;from: Date, to: Date$#125;&gt;</code>
```

Comment:
should be &#125, not $ --> 
![image](https://user-images.githubusercontent.com/16007382/34383861-5ff36fb4-eae6-11e7-8e68-445578832e45.png)